docs/tables/azure_automation_variable.md

```diff
@@ -0,0 +1,52 @@
+# Table: azure_automation_variable
+
+Variable assets are values that are available to all runbooks and DSC configurations in your Automation account. You can manage them from the Azure portal, from PowerShell, within a runbook, or in a DSC configuration.
```

Comment:
@ParthaI could you also add an example wherein the GET hydrate call is being invoked?